PWGHF/TableProducer/candidateCreatorSigmac0plusplusCascade.cxx

```diff
@@ -63,6 +61,9 @@ struct HfCandidateCreatorSigmac0plusplusCascade {
   Configurable<float> softPiDcaZMax{"softPiDcaZMax", 0.065, "Soft pion max dcaZ (cm)"};
   Configurable<bool> addQA{"addQA", true, "Switch for the qa PLOTS"};
 
+  /// Table with Σc0,++ info
+  Produces<aod::HfCandScCasBase> rowScCandidates;
+
```

Comment:
Please see the PWGHF guidelines for the ordering. `Produces` should be the first `struct` member.